setup.py

```diff
@@ -39,7 +41,8 @@
 
 for extra_path in chain([qpdf_source_tree], extra_includes, extra_library_dirs):
     if extra_path and not exists(extra_path):
-        raise FileNotFoundError(extra_path)
+        pass
+        # raise FileNotFoundError(extra_path)
```

Comment:
this part should be reviewed:

for arm64-mac using macports instead of homebrew, this line throws exception due to not having homebrew folders... but it might serve some purpose I might not know about